obsreport/obsreport_exporter.go

```diff
@@ -89,37 +88,37 @@ func (exp *Exporter) createOtelMetrics(cfg ExporterSettings) error {
 		obsmetrics.ExporterPrefix+obsmetrics.SentSpansKey,
 		metric.WithDescription("Number of spans successfully sent to destination."),
 		metric.WithUnit("1"))
-	errors = multierr.Append(errors, err)
+	errors = errors.Join(errors, err)
```

Comment:
Please rename variable `errors` to not clash with the package name

Reply:
`errs`